deb/atlantis-manager/DEBIAN/preinst

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+kill_manager() {
+  echo "triggering maintenance mode"
+  touch /etc/maint
```

Comment:
I think the location is /etc/atlantis/manager/maint for manager.  @edanaher please confirm.